resolwe/flow/managers/workload_connectors/kubernetes.py

```diff
@@ -534,6 +534,8 @@ def start(self, data: Data):
 
         requests["cpu"] = limits.pop("cores")
         limits["cpu"] = requests["cpu"] + 1
+        # Overcommit CPU by 20%.
+        requests["cpu"] *= 0.8
```

Comment:
It would be nice to add this to the changelog.

Reply:
OK, will do.